README.md

```diff
@@ -1,20 +1,31 @@
-<p align="center">
-	<img src="./docs/govpp-logo.png">
-</p>
-<h1 align="center">GoVPP</h1>
+<h1 align="center" style="border-bottom: none">
+    <img alt="Prometheus" src="./docs/govpp-logo.png"><br>GoVPP
```

Comment:
`Prometheus` ?

Reply:
Yes, obviously the god of GoVPP, didn't you know? 😬